patient-db/src/main/java/com/hartwig/hmftools/patientdb/CheckSampleExistsInDatabase.java

```diff
@@ -0,0 +1,57 @@
+package com.hartwig.hmftools.patientdb;
+
+import static com.hartwig.hmftools.common.utils.config.CommonConfig.SAMPLE;
+import static com.hartwig.hmftools.patientdb.dao.DatabaseAccess.addDatabaseCmdLineArgs;
+import static com.hartwig.hmftools.patientdb.dao.DatabaseAccess.databaseAccess;
+
+import java.sql.SQLException;
+
+import com.hartwig.hmftools.patientdb.dao.DatabaseAccess;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jetbrains.annotations.NotNull;
+
+public class CheckSampleExistsInDatabase
+{
+
+    private static final Logger LOGGER = LogManager.getLogger(CheckSampleExistsInDatabase.class);
+
+    public static void main(@NotNull String[] args) throws ParseException, SQLException
+    {
+        Options options = createOptions();
+        CommandLine cmd = new DefaultParser().parse(options, args);
+        String sample = cmd.getOptionValue(SAMPLE);
+
+        try(DatabaseAccess dbAccess = databaseAccess(cmd))
+        {
+            if(dbAccess.readPurpleSampleList().contains(sample))
+            {
+                LOGGER.info("Sample '{}' exists in the database", sample);
+            }
+            else
+            {
+                LOGGER.info("Sample '{}' does not exist in the database", sample);
+                System.exit(2);
```

Comment:
Do you want to exit(2) in case a sample does not exist? It is a normal use-case for a sample not to exist (every first load).

Reply:
I specifically added a special exit code to make this new class easier to use by automated tooling. Checking an exit code is a lot simpler than, for example, capturing and parsing the output.

The value `2` is a bit arbitrary, but aligns with the standard error code for "no such file or directory" ([ENONENT](https://www.baeldung.com/linux/standard-error-eodes)), which seemed a similar case. I wanted to keep `1` to indicate a real error, so `2` seemed logical as "one more option besides 1".

Reply:
Clear. Maybe "invert" the check though, such that an exit code is thrown in the case that a sample does exist, instead of when it does not. I'll approve anyway.